bitcoin-review-podcast/lightning-privacy-splice-panel.md

```diff
@@ -1060,14 +1004,14 @@ So like, you know, there is a scale here that, that we're going to slowly get in
 
 ## Splicing
 
-Speaker 0: 00:54:39
+NVK: 00:54:39
 
 So it sounds like, splicing has both like this amazing privacy aspects to it.
-And then he has very cool scaling things too.
+And then it has very cool scaling things too.
 And also incredible like economic advances too.
-So who wants to explain Spicey?
+So who wants to explain Spicy?
```

Comment:
It's important to pay attention to the whole context of the discussion.
What exactly is "Spicy" here? The title of the chapter as well as the title of the transcript is "splicing".
```suggestion
So who wants to explain splicing?
```